steps/PageSlugs.ts

```diff
@@ -131,9 +133,9 @@ export const repeatingStepSlugs = [
 
 export const urlObjectForSlug = (
   router: NextRouter,
-  slug: string
+  slug: string | undefined
 ): { pathname: string } => {
-  return prefixUrl(router, { pathname: `/${slug}` });
+  return prefixUrl(router, { pathname: slug === undefined ? "" : `/${slug}` });
```

Comment:
`slug` should never be undefined. If it is, resolve that before calling this function.